test/document.spec.js

```diff
@@ -149,24 +185,51 @@ describe('Document', () => {
   it('should validate that users can update details.', (done) => {
     request.put('/api/documents/Another').set('x-access-token', token).set('Accept', 'application/json').send({
       content: 'This is change'
-    }).expect(200).end((err, res) => {
+    })
+    .expect(200)
+    .end((err, res) => {
       expect(res.body.content).to.equal('This is change');
       done();
     });
   });
 
   it('should validate that users can update details.', (done) => {
```

Comment:
### Critical

Please update the title of this test case to be more reflective of what it's actually doing.